### DISCUSSION TITLE
pdm add source code package for debugging (pip -e <package source code path)

### DISCUSSION THREAD

**Discussion by voutsasa** [pdm-project/pdm#2051] [General]
Dear all, 

How could add in a dependency on the source code of a package enabling debugging? 

The pdm add should have the enable the same behaviour as pip -e <package source code path>

Thanks

Achilleas

**Comment by frostming** [comment]
What problem did you meet when running pdm add -e package?

**Comment by voutsasa** [comment]
I get the following error:
```
pdm add -edG dev <package source code path>
Adding group default to lockfile
[PdmUsageError]: Cannot add editables to the default or optional dependency group
```

**Reply by voutsasa** [reply]
Also tried with:
```
pdm add -e <package source code path>
Adding group default to lockfile
[PdmUsageError]: Cannot add editables to the default or optional dependency group
```

**Reply by frostming** [reply]
Install to dev group: `pdm add -d -e <package>`
